src/dotnet-gcmon/dotnet-gcmon.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0;net5.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Since it is working with .NET Core 3.1, we should only build with it: it will work on 3.1, 5.0, and 6.0 and not only 6.0.
BTW, this is what is done for dotnet-counters (https://github.com/dotnet/diagnostics/blob/main/src/Tools/dotnet-counters/dotnet-counters.csproj#L4)

In addition, please update the <PackageVersion> to 0.3.0 to ensure that nuget will see the new package associated to a higher version as the currently uploaded.

Reply:
agree with @chrisnas - I noticed that it was building all 3 versions which seems unnecessary.

Reply:
Done.